### DISCUSSION TITLE
Custom domain and verifier update(too urgent)

### DISCUSSION THREAD

**Discussion by infocodiste** [Web3Auth/Web3Auth#637] [Help]
I am using the web3Auth + auth0.
When I created the verifier for the mainnet, I used the standard default domain of auth0 to create the verifier. But now, I have created the custom domain in auth0. So, do I need to update the verifier as well. If yes, then will the changes be applied as soon as I change the domain in verifier or it well take some time?

**Comment by rinzler99** [comment]
Hey, Please update your verifier and let us know if it isn't deployed soon.

**Reply by rinzler99** [reply]
Is your verifier still stucked at deploy? Please let us know.
